docs/streams/developer-guide/config-streams.html

```diff
@@ -778,10 +778,21 @@ <h4><a class="toc-backref" href="#id34">rack.aware.assignment.tags</a><a class="
           <span id="streams-developer-guide-max-warmup-replicas"></span><h4><a class="toc-backref" href="#id29">max.warmup.replicas</a><a class="headerlink" href="#max-warmup-replicas" title="Permalink to this headline"></a></h4>
           <blockquote>
             <div>
-              The maximum number of warmup replicas (extra standbys beyond the configured num.standbys) that can be assigned at once for the purpose of keeping
-              the task available on one instance while it is warming up on another instance it has been reassigned to. Used to throttle how much extra broker
-              traffic and cluster state can be used for high availability. Increasing this will allow Streams to warm up more tasks at once, speeding up the time
-              for the reassigned warmups to restore sufficient state for them to be transitioned to active tasks. Must be at least 1.
+              <p>
+                The maximum number of warmup replicas (extra standbys beyond the configured num.standbys) that can be assigned at once for the purpose of keeping
+                the task available on one instance while it is warming up on another instance it has been reassigned to. Used to throttle how much extra broker
+                traffic and cluster state can be used for high availability. Increasing this will allow Streams to warm up more tasks at once, speeding up the time
+                for the reassigned warmups to restore sufficient state for them to be transitioned to active tasks. Must be at least 1.
+              </p>
+              <p>
+                Note that one warmup replica corresponds to one Stream Task. Furthermore, note that each warmup replica can only be promoted to an active Task during
```

Comment:
@ableegoldman could you please confirm this?

@mjsax any chance you could also let me know how to preview the changes to the HTML? I tried opening the file in my browser it wouldn't render...this is somewhat embarrassingly my first time editing HTML.

Reply:
@littlehorse-eng You need apache http with a include_module and rewrite_module being loaded. Maybe this can help: https://github.com/lucasbru/kafka-site-docker

Reply:
"Stream Task" sound a little bit like "active task" (at least, we use `StreamTask.java` in the code...)

Maybe:
```
Note: in contrast to <code>num.standbys</code> that works similar to topic <code>replication.factor</code> config, <code>max-warmup-replicas</code> is the <it>absolute</it> numbers of warmup tasks.
```